legend-engine-pure-code-compiled-core/src/main/resources/core/legend/service/metamodel.pure

```diff
@@ -25,9 +25,9 @@ Class meta::legend::service::metamodel::Service extends PackageableElement, meta
    multiExecutionKeyMustBePartOfPattern: if($this.execution->instanceOf(meta::legend::service::metamodel::PureMultiExecution) && $this.execution->cast(@meta::legend::service::metamodel::PureMultiExecution).executionParameters->isNotEmpty() ,|$this.pattern->contains('{'+ $this.execution->cast(@PureMultiExecution).executionKey+'}'),|true),
 
    executionAndTestTypesMatch: if($this.execution->instanceOf(meta::legend::service::metamodel::PureMultiExecution),
-                                  | assert($this.test->instanceOf(MultiExecutionTest),'Service with multiple executions requires a MultiExecution test'),
+                                  |if($this.test->isNotEmpty(),| assert($this.test->toOne()->instanceOf(MultiExecutionTest),'Service with multiple executions requires a MultiExecution test'),| true),
```

Comment:
could we do an && rather than an if on the assert?

`assert($this.test->isNotEmpty() && ...)`

or you could just do map to execute whenever it exists:

`this.test->map(a | assert(...))`

Reply:
ended up going with assert($this.test->isNotEmpty() && ...)